src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/osd/osd-performance-histogram/osd-performance-histogram.component.scss

```
@@ -1,10 +1,15 @@
+$table-components-size: 10px;
```

I think cell is a more appropriate name:
```suggestion
$table-cell-size: 10px;
```